examples/step-33/step-33.cc

```diff
@@ -1607,7 +1607,11 @@ namespace Step33
       }
 
     // After all this assembling, notify the Trilinos matrix and vector object
-    // that the linear system is done:
+    // that the linear system is done by invoke the function compress(). However,
+    // these two function calls here have no actual influence and you can remove
+    // them safely. The reason is this tutorial program is not designed to run in
+    // distributed parallel mode. See @ref GlossCompress "Compressing distributed objects"
+    // for more information on what is compress() designed to do.
```

Comment:
Maybe. On the other hand, this is a tutorial where we teach things that haven't been discussed in other programs. It's also the case that if you forget to call `compress()` on the matrix, you will quickly realize that something is wrong -- so if you remove both calls, I think few people will only add one of the calls, not both.

In the end, it's not an important point. I'll defer to others.